### DISCUSSION TITLE
Common issues of MONARC usage

### DISCUSSION THREAD

**Discussion by ruslanbaidan** [monarc-project/MonarcAppFO#392] [Q&A] [answered]
1. _"File missing" error during the import._
  **Reason**: Importing file is bigger than allowed by the php.ini server configuration.
  **Solution**: Change the php.ini settings, described [here](https://github.com/monarc-project/MonarcAppFO/blob/master/vagrant/bootstrap.sh#L22-L26) and restart the apache service.

**Comment by cedricbonhomme** [comment, marked as answer]
added [here](https://www.monarc.lu/documentation/common-issues/#file-missing-error-during-the-import) ;-)
